### ISSUE TITLE
`hiddenColumns` plugin unsets cell's `renderer`

### ISSUE THREAD

**Issue by bob-frost** [handsontable/handsontable#5883]
### Description
Name of original renderer can't be obtained with `getCellMeta(row, col).baseRenderer` or `getCellMeta(row, col).renderer` if `hiddenColumns` plugin is enabled and column is not hidden initially.
As far as I understood `hiddenColumns` should copy original renderer to `baseRenderer` property of meta when column is hidden and restore it back to `renderer` when column is displayed. Instead `renderer` becomes `undefined` and `baseRenderer` is `null`
Seems that issue is on this line: https://github.com/handsontable/handsontable-pro/blob/938ac9d34382320b313e6cd8dd103474eaa7d878/src/plugins/hiddenColumns/hiddenColumns.js#L255 Condition should be `else if (!cellProperties.baseRenderer)` because initially `baseRenderer` is not `null` but `undefined`.

### Steps to reproduce
See console in the jsfiddle demo

### Demo
https://jsfiddle.net/bob_usenakunov/L46xqezg/7/

### Your environment
* Handsontable version: 3.0.0
* Browser Name and version: Chrome 66.0.3359.181
* Operating System: MacOS

**Comment by AMBudnik** [comment]
> @AMBudnik I have few custom renderers and need to be able to identify them.
> Without hiddenColumns plugin there was smth like this:
> 
> const isReactCell = (hotInstance, row, col) => {
>   const renderer = hotInstance.getCellMeta(row, col).renderer;
>   return renderer && renderer.startsWith('react');
> };
> After adding hiddenColumns to table there is no way to get renderer name from cell meta. I found another way to do it:
> 
> const isReactCell = (hotInstance, row, col) => {
>   const renderer = hotInstance.getSettings().columns[col].renderer;
>   return renderer && renderer.startsWith('react');
> };
> But it seems that it would be more proper way to keep renderer accessible from cell meta

**Comment by AMBudnik** [comment]
Update for version `7.1.1`

I've been provided with a great example that also shows this issue, and after simplifying I got https://jsfiddle.net/handsoncode/xyqfeaud/

![719afb95f3776afd389b3abd58fcb9069427f319](https://user-images.githubusercontent.com/10757813/65862955-93ad6480-e36f-11e9-9aa6-6f74e52b4e5a.png)

The behavior doesn't differ if we define hiddenColumns or not. All the cases below doesn't change a thing.

```js
 hiddenColumns: true
```

```js
  hiddenColumns: {}
```

```js
  hiddenColumns: {
    columns: [],
    indicators: true
  }
```

```js
 hiddenColumns: {
    columns: [1],
    indicators: true
  }
```

### When the issue’s gone
* when we comment our `colWidths`
* when we comment our `hiddenColumns`
* when we call `render` method after initializing the table
* when we set  `hiddenColumns: false`

### Environment
browser: Chrome 77
device: Intel Core i7-8565U CPU 1.80GHz
handsontable: `7.1.1.`. Works the same in older versions and builds for `8.0.0.-rc1`

**Comment by warpech** [comment]
I have another, simple reproduction of this problem: https://jsfiddle.net/0sp91x6n/

##### Expected

All cells should be rendered using the custom renderer that writes `:)` into a cell

##### Actual

Some cells are rendered using the default renderer, especially on the first render and when fast scrolling.

**Comment by wojciechczerniak** [comment]
@Osky772?  👀

**Comment by Osky772** [comment]
It was closed accidentally due to GitHub. In PR https://github.com/handsontable/spreadsheet-viewer-demo/pull/219 I wrote the word "resolved" and then I put a link to this issue in Handsontable - GitHub decided to automatically close this issue. Sorry about that.

**Comment by AMBudnik** [comment]
### Main case

Solved in 8.0.0-beta2-rev2 https://jsfiddle.net/w374njke/

![](https://user-images.githubusercontent.com/10757813/81396442-b97cc200-9125-11ea-8294-1f062d1a9d46.png)

##### @bob-frost  please confirm

### @AMBudnik case

Solved in 8.0.0-beta2-rev2 https://jsfiddle.net/nvouh742/

### @warpech case

**Solved** with 8.0.0.-beta2-rev2 https://jsfiddle.net/j4apfnLb/

**Comment by AMBudnik** [comment]
Hi @bob-frost @slavabogov

I have good news! We've just released 8.0.0-beta2 that fixes the mentioned issue. We won't be closing this ticket now. It will be closed after we release the final 8.0.0 version of Handsontable.

Here’s a full list of changes https://github.com/handsontable/handsontable/releases/tag/8.0.0-beta.2
NPM https://www.npmjs.com/package/handsontable/v/8.0.0-beta.2 

All feedback's appreciated. Thank you for your input.

**Comment by AMBudnik** [comment]
Hi @bob-frost 

we have an update for Handsontable v 8.0.0, released before the weekend.

The main case - https://jsfiddle.net/3ec7z8k6/- fixed
My case - https://jsfiddle.net/5x1esft7/ - fixed
@warpech case - https://jsfiddle.net/hbsfxugk/ - fixed

Please confirm if it works in your project as well.
